pipeline_dp/combiners.py

```diff
@@ -220,6 +228,9 @@ def compute_metrics(self, accumulator: AccumulatorType) -> dict:
     def metrics_names(self) -> List[str]:
         return ['privacy_id_count']
 
+    def explain_computation(self) -> str:
```

Comment:
A general comment, not just for this string: can we make this more friendly towards non-experts in DP?

For example, instead of “applied truncated geometric”, can we write something like “for every partition, calculate counts of privacy IDs, add <x> Gaussian noise, and drop partitions where the noised count is less than <x>”

Similarly, instead of “computed privacy id count with (eps=eps0)”, can we write something like “Computed privacy ID and add Gaussian noise of scale <x>”

also for the others.